blazor/datagrid/context-menu.md

```diff
@@ -392,4 +392,62 @@ The following sample code demonstrates how to enable or disable context menu ite
 }
 ```
 
-N> You can refer to [Blazor DataGrid](https://www.syncfusion.com/blazor-components/blazor-datagrid) feature tour page for its groundbreaking feature representations. You can also explore [Blazor DataGrid example](https://blazor.syncfusion.com/demos/datagrid/overview?theme=bootstrap4) to understand how to present and manipulate data.
\ No newline at end of file
+N> You can refer to [Blazor DataGrid](https://www.syncfusion.com/blazor-components/blazor-datagrid) feature tour page for its groundbreaking feature representations. You can also explore [Blazor DataGrid example](https://blazor.syncfusion.com/demos/datagrid/overview?theme=bootstrap4) to understand how to present and manipulate data.
+
+### Get specific row details in the contextmenu click event
+
+This section provides information on how to retrieve specific row details in the context menu click event using Syncfusion Grid properties, methods, and events.
```

Comment:
you can obtain the row details using the RowInfo argument of ContextMenuClicked event.